CefSharp/DevTools/DevToolsClient.cs

```diff
@@ -80,16 +80,10 @@ public void SetDevToolsObserverRegistration(IRegistration devToolsRegistration)
         /// <inheritdoc/>
         public void AddEventHandler<T>(string eventName, EventHandler<T> eventHandler) where T : EventArgs
         {
-            if (eventHandlers.TryGetValue(eventName, out IEventProxy eventProxy))
-            {
-                var p = ((EventProxy<T>)eventProxy);
-                p.AddHandler(eventHandler);
-            }
-            else
-            {
-                eventProxy = new EventProxy<T>(eventHandler, DeserializeJsonEvent<T>);
-                eventHandlers.TryAdd(eventName, eventProxy);
-            }
+            var eventProxy = eventHandlers.GetOrAdd(eventName, _ => new EventProxy<T>(eventHandler, DeserializeJsonEvent<T>));
```

Comment:
Please note that in the demo code I have removed the `eventHandler` parameter from the constructor so that it isn't added twice.

Reply:
Thanks.  The code didn't compile. EventProxy also needs to be updated.